tensorflow/core/public/version.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 #define TF_MAJOR_VERSION 0
 #define TF_MINOR_VERSION 12
-#define TF_PATCH_VERSION head
+#define TF_PATCH_VERSION 0-rc0
```

Comment:
yea, we change the version test in this PR to check that we have a '-' before rc, which 'head' or 'post' will break. I'll need to update both.